GOVERNANCE.md

```diff
@@ -101,21 +93,20 @@ Advisory Committee members:
 In the event of any conflict of interest (a Committee Member, their family member, or someone with whom the Committee Member has a close academic or employment relationship is involved in a decision), the Committee Member must immediately notify other Committee Members. The Committee Member will be asked to recuse themselves from ongoing conversations, and decision process regarding the Transaction.  
 
 #### Current list of Advisory Committee members
-* Martin Magdinier
 * Jan Ainali
 * Antonin Delpeuch
+* [Open Position]
 
-### Project Director
-OpenRefine's Project Director (paid position) works closely with OpenRefine’s advisory and steering committees, and with its communities of users and contributors.
+### Project Manager
+OpenRefine's Project Manager (paid position) works closely with OpenRefine’s advisory and steering committees, and with its communities of users and contributors.
```

Comment:
This still mentions the steering committee.

Reply:
As we are transitioning to a new format still to be defined, I left the mention of the Steering Committee and indicated that `There is currently no Steering Committee`
If the group think it is confusing, we can remove the mention of the Steering Committee entirely from this document

Reply:
Perhaps it would make more sense to settle on a governance structure and **then** update the documentation describing it.

I'm still unclear on what happened to the Steering Committee. Did they vote themselves out of existence? Were they disbanded by some other party(ies)? If so, who?

Reply:
They just stopped meeting regularly.

I think it's worth making small updates to this document like in this PR and still work on bolder changes afterwards. I intend to post proposals on the forum about this soon.